RickLeinecker/Fortuna
# Frontend 3v3 handling 1/2

**Description**
Sets up the look for frontend 3v3 battle handling. Needs more work to be finished. Currently, this creates the button to toggle 3v3 display and the 1v1 display in Battle Arena and Training Arena. I had to make edits for the SelectTank and TankDisplay components to get this to function correctly.

I have edited the prepareMatchAPICall to have one for 3v3 and one for 1v1 as well.

**Resolves These Issues**
Resolves none yet

**Type of change**
Check options that are relevant.


**Steps to Verify Functionality**
1. Go to battle arena.
2. Switch between 1v1 and 3v3.
3. Remove all selected tanks.
4. Try to challenge someone.
5. View toast error.
6. Go to 1v1 and ensure battles still work.
7. Go to training arena.
8. Switch between 1v1 and 3v3.
9. Remove all bot tanks and start battle.
10. View toast.
11. Remove all user tanks and start battle.
12. View different toast.
13. Go to 1v1 and ensure battles still work.
14. Go to armory to ensure the new TankDisplay and SelectTank haven't destroyed functionality.

**Final Checklist:**
Go down the list and check them off.
